kubernetes/config-maps/nginx/ssl.conf

```diff
@@ -0,0 +1,26 @@
+# Disable SSLv3
+ssl_protocols              TLSv1 TLSv1.1 TLSv1.2;
+
+# Enable server-side protection
+ssl_prefer_server_ciphers  on;
+ssl_ciphers                "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
```

Comment:
Please add some comments explaining why you chose those specific ciphers. Best if you can add a link to some authoritative recommendation.

Reply:
This generator from Mozilla can give you some good values: https://mozilla.github.io/server-side-tls/ssl-config-generator/

Reply:
Also, see this: https://cipherli.st/

Reply:
Done.

Reply:
I use value from https://cipherli.st/